packages/product-feed-google/src/Migrations/Version20230405111441.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ProductFeed\GoogleBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20230405111441 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $seoRobotsContentSettingCount = $this->sql(
+            'SELECT COUNT(*) FROM setting_values WHERE name = \'mailWhitelist\' AND domain_id = 1;'
```

Comment:
I think it's not necessary to check for the presence of the setting value first... if the same already exists, it may not be compatible with our code, so it's potentially dangerous. 

It's always possible to skip this migration in the project.

Reply:
Removed.